pkg/util/config/feature_config.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/spf13/pflag"
+)
+
+const (
+	// known keys
+	enableAllAlphaFeatures = "enableAllAlphaFeatures"
```

Comment:
naming: starting this with "enable" implies that some of these might be "disable" flags.  I think we should make them all the same polarity, even if some have different defaults.  Given that "allAlphaFeatures" is better, or even "allAlpha", or even "alpha".  

That said, for symbol names, a common prefix helps sort, so "enAlpha" or "enableAlpha".  Now, if individual alpha features have their own gates, who trumps?